tests/data/stac_item_testdata_xclim_cmip6_ncml.json

```diff
@@ -93,14 +93,14 @@
       ]
     },
     "WCS": {
-      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/wcs/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc?service=WCS&version=1.0.0&request=GetCapabilities",
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/wcs/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc",
```

Comment:
WMS and WCS access URLs themselves don't contain any additional parameters. 

Note, the THREDDS catalog browser will append these to the link when accessing the service through the HTML interface but the XML version does not contain the additional parameters (the XML data is what is used to populate STAC).